test/runtests.jl

```diff
@@ -29,6 +29,7 @@ end
     @test Int64(BFloat16(10)) == Int64(10)
     @test BFloat16(BigFloat(1)) == BFloat16(1)
     @test BigFloat(BFloat16(1)) == BigFloat(1)
+    @test BFloat16(pi) == BFloat16(3.14159)
```

Comment:
```suggestion
    @test BFloat16(pi) == BFloat16(3.14159)
    @test all(R -> R<:BFloat16, Base.return_types(BFloat16))
```
Test adapted from https://github.com/JuliaLang/julia/pull/37213

Reply:
@kengruven Did you miss this test suggestion?

Reply:
I was going to look into this further, because I'm new to Julia and don't understand this syntax, but if you say it's OK then I'm OK with it.